op/framework.py

```diff
@@ -372,7 +373,7 @@ def __init__(self, data_path, charm_dir, meta, model):
         self.model = model
         self._event_count = 0
         self._observers = []      # [(observer, method_name, parent_path, event_key)]
```

Comment:
Hmmm.. won't the observer only get into the wakref map if it was also added here?

Reply:
Reminder from the call that the comment needs to be updated because the first element of each tuple is actually the observer path rather than the instance.

Reply:
Thanks. I had done the change, but forgot to push.